src/components/InlineDirectoryPicker.vue

```diff
@@ -22,12 +22,13 @@ export default {
     return {
       browse: false,
       browsingPath: null,
-      browsingTree: null
+      browsingTree: null,
+      separator: this.$core.config.get('pathSeparator')
```

Comment:
I would have used the same name for this property in the data object (`pathSeparator`) to make it consistent.

Reply:
Yes indeed, fixing this